### ISSUE TITLE
FEATURE: Pass in seach engine

### ISSUE THREAD

**Issue by AethanFoot** [manilarome/rofi-spotlight#4]
I would be awesome if you could pass the search engine when you call rofi, this would be helpful if you say bind super g to open global search with the search engine as google, this then could possibly be expanded to allow users add custom search engines (I don't if this would even be possible). A different option might be to add a flag along with :w where you can set :wg to search in google.

**Comment by eromatiya** [comment]
Uhm you can just change your default search engine in the python script.

**Comment by AethanFoot** [comment]
It was more to be able to have multiple search engines setup at once and to chose which  one you use at the time, which then for example would allow you to query the archwiki or the urban dictionary instead of going through google then to the sites. The first part was more to allow for easy keybinding, say mod g for google and mod d for duckduckgo.

**Comment by eromatiya** [comment]
Yeah, sounds good! The problem is I don't know if I can implement an autosuggestion for those(archwiki, urban dictionary, etc.). But we can easily add a shortcut tho, for example <kbd>:wd</kbd> for duckduckgo and etc.
